assets/src/dashboard/app/views/myStories/content/storyGridItem/components.js

```diff
@@ -128,19 +128,24 @@ export const CardDetailsColumn = styled.div`
 // If it fails, max-height will catch so that really long titles
 // don't leave their designated space, just no ellipsis.
 export const Title = styled(Headline).attrs({
-  as: 'a',
   size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
 })`
   overflow: hidden;
   text-overflow: ellipsis;
-  color: ${({ theme }) => theme.colors.inverted.fg.primary};
+  color: ${({ theme }) => theme.colors.inverted.fg.tertiary};
```

Comment:
Not 100% sure about this change. Perhaps we might just want to keep this primary at all times.

Reply:
I'd like to leave this as primary and then just adjust the cursor instead. If it's an anchor tag it should show the `pointer` and if it's just a headline it should be `default` - which i believe is how it's set up since you separated the styled component based on if it's got a link or not.

Reply:
I originally had this way and there was no way to see it wasn't a link. As a user, I would just try and click on the link and then nothing happens, which is confusing. Here is what it looks like. 

<img width="1277" alt="Screenshot 2021-08-09 at 19 07 13" src="https://user-images.githubusercontent.com/237508/128753067-9ec9a9d7-6aa5-4070-b6a7-1d61734cd644.png">

I would also say, the title's isn't super clear it is a link in the first place. This makes it more confusing. 

We also have to compare it to WordPress, where the links are a different colour. 

<img width="379" alt="Screenshot 2021-08-09 at 19 09 40" src="https://user-images.githubusercontent.com/237508/128753309-e5878b23-bb20-41ad-b4ae-f4f0ef4e468f.png">

Can we have some styling the denote this is not a link?

Reply:
Changing color decreases contrast quite a bit, especially when there is no poster image, so it's grey on dark grey. That's also not ideal.

I think we should stick with primary for now & create a new ticket to come up with something suitable if needed.

Reply:
well, really the link as a title is a relic of the old structure and you can open the link in the context menu just fine. 

so maybe we just cut to the chase and make it a plain ol title and you use the menu to open a story.

Reply:
I have created a breakout issue https://github.com/google/web-stories-wp/issues/8652